apps/gateway/.env.example

```
@@ -0,0 +1,35 @@
+QSTASH_URL=
+QSTASH_CURRENT_SIGNING_KEY=
+QSTASH_NEXT_SIGNING_KEY=
+
+# Service Auth
+GATEWAY_API_KEY=              # Console→Gateway authentication
+GATEWAY_WEBHOOK_SECRET=       # Gateway→Console HMAC signing
+ENCRYPTION_KEY=               # AES-256-GCM for token encryption (32+ hex chars)
+
+# GitHub
+GITHUB_APP_SLUG=
+GITHUB_APP_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_APP_ID key should go before the GITHUB_APP_SLUG key

(UnorderedKey)